app/src/main/java/co/nimblehq/compose/crypto/ui/screens/compose/home/PortfolioCard.kt

```diff
@@ -0,0 +1,131 @@
+package co.nimblehq.compose.crypto.ui.screens.compose.home
```

Comment:
This project will be a full Jetpack Compose project. Should we consider to also skip `compose` sub package? This is my new suggested package structure, how do you think? @Wadeewee @hoangnguyen92dn 

![image](https://user-images.githubusercontent.com/16315358/178887794-aadca1b5-c014-4140-ad0e-186109aed0be.png)

Reply:
Good point, @luongvo 👍 and Yeah, I agree we should skip `compose` sub package, I will update asap 🚀